tools/local-presubmit

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# A basic script to locally semi-simulate the build and test
+# operations performed by our continuous integration service. Note
+# that this does not do a clean and reinstall of npm modules and
+# sigh/bazel artifacts.
+
+MAG="\033[95m"
+END="\033[0m"
+
+status() {
+    echo -e "$MAG$1$END"
+}
+
+ROOT=$(dirname $0)/..
+cd $ROOT
+status "Testing all of the things."
+
+./tools/sigh && \
+    bazel test //src_kt/... //src/... //particles/... && \
+    ./tools/sigh testShells && \
+    ./tools/ktlint && \
+    ./tools/sigh lint && \
```

Comment:
If you add `set -e` to the top of the file, then these can all just go on their own lines and you won't need `&&`. (`set-e` stops the script if any of the commands return an error)

Reply:
Good tip.

Reply:
Thanks! I learned something new. Done.